DAQ/src/StrawHitRecoFromFragments_module.cc

```diff
@@ -33,6 +33,7 @@
 #include "Offline/RecoDataProducts/inc/StrawDigi.hh"
 #include "Offline/RecoDataProducts/inc/ComboHit.hh"
 #include "Offline/RecoDataProducts/inc/StrawHit.hh"
+#include "Offline/DataProducts/inc/EventWindowMarker.hh"
```

Comment:
I think we should use the `mu2e::EventHeader`: https://github.com/Mu2e/artdaq_core_mu2e/blob/develop/artdaq-core-mu2e/Data/EventHeader.hh

Reply:
I think having a separate object (EventWindowMarker) as part of the Offline provides some necessary buffering from the Online EventHeader, just as Offline digis provide buffering from fragments.   In particular, this simplifies the simulation workflow.  Creating an EventWindowMarker from an EventHeader should be easy, and can be done in a future PR.